charts/kube-state-metrics/templates/role.yaml

```diff
@@ -1,180 +1,192 @@
-{{- if .Values.rbac.create -}}
+{{- if and (eq $.Values.rbac.create true) (not .Values.rbac.useExistingRole)  -}}
+{{- if eq .Values.rbac.useClusterRole false }}
+{{-   range (split "," $.Values.namespace) }}
```

Comment:
```suggestion
{{-   range (split "," $.Values.namespaces) }}
```

Reply:
You are absolutely correct Jianhao. (namespace was before 2.0 - its called namespaces in 2.0+). This chart has since been moved to https://github.com/prometheus-community/helm-charts/blob/d6ae19c6bb7089bd0080952ca6f98b68177e3dcb/charts/kube-state-metrics/templates/role.yaml#L3 - where it still lists namespace instead of namespaces - even though it has moved to 2.0.